.storybook/preview.tsx

```diff
@@ -58,25 +57,6 @@ const customViewports = {
 const MyDocsContainer = (props) => (
   <MDXProvider
     components={{
-      h1: ({ children }) => (
```

Comment:
With the update in `.storybook/preview-body.html`, we can remove the DS overrides. They were mostly for styling but it previously did not allow us to use the id hash linking from headings. This is now done automatically by Storybook in their links and headings and the update in `.storybook/preview-body.html` just updates the font family for consistent styling.